lib/Config.js

```diff
@@ -82,6 +82,9 @@ class Config extends EventEmitter {
             this.bootstrapList = [];
         }
 
+        if (process.env.SITE_NAME) {
+            this.bootstrapList = this.bootstrapList.filter(item => item.site === process.env.SITE_NAME);
+        }
```

Comment:
Accessing process.env variable is slow in general (it executes a C code at the system level), in general we tend to avoid accessing the same env variable multiple times:

```suggestion
        const shouldRestrictToSite = process.env.SITE_NAME;
        if (!!shouldRestrictToSite) {
            this.bootstrapList = this.bootstrapList.filter(item => item.site === shouldRestrictToSite);
        }
```